README.md

```diff
@@ -85,7 +85,7 @@ Clone this repository to your desired folder:
 👤 **Emeka Emmanuel**
 
 - GitHub: [@emekaemmanuel](https://github.com/emekaemmanuel)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/emeka-ugboaja-167820226)
+- LinkedIn: [Emeka Ugboaja](https://linkedin.com/in/emeka-ugboaja-167820226)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p> 
```

Comment:
- [ ] Please in Desktop version the hello part is not presented in the design 
your design 

![image](https://user-images.githubusercontent.com/74744787/210814735-cfcc0836-2dd9-4358-8591-a7cbf9f403ab.png)

behnace 
![image](https://user-images.githubusercontent.com/74744787/210814950-6fb3e074-1dc4-4283-b50a-ebe44c724bd5.png)

Reply:
✔️